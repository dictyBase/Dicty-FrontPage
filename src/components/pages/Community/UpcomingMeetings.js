// @flow
import React from "react"
import Grid from "@material-ui/core/Grid"
import FontAwesome from "react-fontawesome"
import { Banner, Header } from "styles/EditablePageStyles"

/**
 * This displays upcoming Dicty meetings.
 */

const UpcomingMeetings = () => (
  <div>
    <Banner>
      <Header>
        <FontAwesome name="plane" />
        <br />
        Upcoming Meetings
      </Header>
    </Banner>
    <br />
    <Grid container justify="center">
      <Grid item>Coming soon...</Grid>
    </Grid>
  </div>
)

export default UpcomingMeetings
