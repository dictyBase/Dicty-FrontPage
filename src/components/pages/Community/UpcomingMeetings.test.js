import React from "react"
import toJson from "enzyme-to-json"
import { shallow } from "enzyme"
import "jest-styled-components"
import UpcomingMeetings from "./UpcomingMeetings"

test("matching a snapshot of JBrowse", () => {
  const subject = shallow(<UpcomingMeetings />)
  expect(toJson(subject)).toMatchSnapshot()
})
