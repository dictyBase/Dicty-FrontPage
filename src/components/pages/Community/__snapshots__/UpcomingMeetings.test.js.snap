// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot of JBrowse 1`] = `
<div>
  <styled.div>
    <styled.h1>
      <FontAwesome
        name="plane"
      />
      <br />
      Upcoming Meetings
    </styled.h1>
  </styled.div>
  <br />
  <WithStyles(Grid)
    container={true}
    justify="center"
  >
    <WithStyles(Grid)
      item={true}
    >
      Coming soon...
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</div>
`;
